src/terra-dev-site/guide/CreatingItemCollection.f.guide.mdx

```diff
@@ -0,0 +1,201 @@
+import ResponsiveListandTableExample from '../example/ResponsiveListAndTableExample?dev-site-example';
```

Comment:
maybe we should name this file `RecreatingItemCollection`? are we ok with this guide being a first level side nav item? it seems odd next to our general guides, such as `general guidelines`,  `installing components`, and `basic usage`. 

if anything, maybe we make this the last item. we could also create an upper level called `Recreating deprecated components`, and place this guide under there. thoughts?

Reply:
Renamed and reorganized here https://github.com/cerner/terra-ui/pull/310/commits/d5f3de15100e9fcb4462f0fa847dc580aa673868#diff-f306b40c35b7837e14535b95656559fca5a16191b1c2d2c2d535b3e07e66705aL1